Sources/RoktUXHelper/Data/Model/CustomStateMap.swift

```diff
@@ -24,7 +24,7 @@ extension CustomStateMap {
 
 public typealias CustomStateMap = [CustomStateIdentifiable: Int]
 
-public struct CustomStateIdentifiable: Hashable {
+public struct CustomStateIdentifiable: Hashable, Encodable, Decodable {
```

Comment:
nit: can use `Codable` its definition is `typealias Codable = Decodable & Encodable`